properties_pane/field_level/fieldLevelConfig.json

```diff
@@ -1388,10 +1398,70 @@ making sure that you maintain a proper JSON format.
 							}
 						},
 						{
-							"propertyName": "Cardinality & nullability",
-							"propertyKeyword": "required",
-							"propertyType": "select",
-							"options": ["<Type>", "<Type>!", "[<Type>]", "[<Type>]!", "[<Type>!]", "[<Type>!]!"]
+							"propertyName": "List item",
+							"propertyKeyword": "listItems",
+							"propertyTooltip": "Add List item",
+							"groupItemLimit": 1,
+							"propertyType": "group",
+							"structure": [
+								{
+									"propertyName": "Required",
+									"propertyKeyword": "required",
+									"propertyType": "checkbox"
+								},
+								{
+									"propertyName": "Type",
+									"propertyKeyword": "type",
+									"propertyTooltip": "Select List item type",
+									"propertyType": "selecthashed",
+									"template": "definitions",
+									"validation": {
+										"required": true
+									},
+									"templateOptions": {
+										"filter": {
+											"condition": {
+												"type": "or",
+												"values": [
+													{
+														"key": "childType",
+														"value": "scalar"
+													},
+													{
+														"key": "childType",
+														"value": "enum"
+													},
+													{
+														"key": "childType",
+														"value": "input"
+													}
+												]
+											},
+											"options": {
+												"useAllDefinitionsAsSource": true,
+												"groupBy": "childType",
+												"includeTargetTypes": {
+													"excludeTypes": [
+														"List",
```

Comment:
The `List` type is excluded from List items to limit the nesting of List types - because we can't use recursive configurations. According to the graphql spec it's supported - but looks like it is mostly an edge case. If we will need to add nested levels we have to discuss the number of levels that we should support. At this stage, we support only one level of list items.